k8s/ueransim/network_test.py

```diff
@@ -0,0 +1,104 @@
+import sys
+from scapy.all import (
```

Comment:
I imagine you already validated this but there isn't an equivalent package in go?